cms/static/js/features/import/factories/import.js

```diff
@@ -35,7 +35,7 @@ define([
                     var filepath = $(this).val(),
                         msg;
 
-                    if (filepath.substr(filepath.length - 6, 6) === 'tar.gz') {
+                    if (/\.tar.gz$|\.zip$/.test(filepath)) {
```

Comment:
question: ❓ should I add a global configuration for this like python code?

or maybe keep this regex defined in the top of the file for reuse?